pymodbus/payload.py

```diff
@@ -330,7 +330,7 @@ def _unpack_words(self, handle) -> bytes:
         # Change Word order if little endian word order  #
         # Pack values back based on correct byte order   #
         # ---------------------------------------------- #
-        :param fstring:
+
```

Comment:
Good catch, except this is an internal function, where we tend not to document the parameters.